### PR TITLE
WIP: Authentication Added ( Login/Signup )

### PR DESCRIPTION
**Pull request by way2nizam**
screenshot-login
screenshot-signup

### CODE REVIEW

File: components/login.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+            <a href="./signup.html">Create New Account >></a>
+        </div>
+    </div>
+
+
+
+</body>
+
+</html>
```
Comment (KanekarSnehal) [inline components/login.html:85]: add new line at the end of code file
Refer here: https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log

File: components/signup.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+            <a href="./login.html">Already have an account >></a>
+        </div>
+    </div>
+
+
+
+</body>
+
+</html>
```
Comment (KanekarSnehal) [inline components/signup.html:82]: same here also add new line

File: styles/login.css [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  height: 90vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: var(--color-grey-light-3);
+}
+
+.login {
+  font-size: 1.5rem;
+  color: #000;
```
Comment (KanekarSnehal) [inline styles/login.css:11]: you can use css variables for color

File: styles/login.css [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  max-width: 52.5rem;
+  max-height: 48rem;
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
```
Comment (KanekarSnehal) [inline styles/login.css:16]: here also you can use css variables

**Review by KanekarSnehal — COMMENTED** [review]
Great work, code looks good.

Suggestion: code formatting needed in html files for consistent new line between two div elements.

File: components/login.html (hunk 1 above)
Comment (way2nizam) [reply components/login.html:85]: learn this new thing, Thanku Kanekar.

File: styles/login.css (hunk 4 above)
Comment (way2nizam) [reply styles/login.css:16]: okay.

**Review by amarphule — COMMENTED** [review]
Looks good @way2nizam

**Pull request merged**